docs/iaas/architecture/networking.md

```diff
@@ -2,8 +2,15 @@
 
 ## Overview
 
-IronCore's networking architecture is designed to provide a robust and flexible networking solution. It consists of 
-several components that work together to ensure network out and inbound connectivity and isolation for `Machine` instances.
+IronCore's virtual networking architecture provides an end-to-end virtual networking solution for virtual machines running in data centers. It is designed to enable robust, flexible and performing networking control plane and data plane.
```

Comment:
I would rather stick here to the `Machine` instances term as in the future we also want to include bare metal servers into such an SDN.